core/iwasm/aot/aot_runtime.h

```diff
@@ -127,6 +132,9 @@ typedef struct AOTModuleInstanceExtra {
     DefPointer(uint8 *, shared_heap_base_addr_adj);
     MemBound shared_heap_start_off;
 
+    DefPointer(AOTGlobalInstance *, globals);
```

Comment:
Had better also add `bh_static_assert(offsetof(AOTModuleInstanceExtra, globals) == 24);` in aot_runtime.c, like the `bh_static_assert` for `shared_heap_start_off`:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/aot/aot_runtime.c#L62